lib/main_containers.pm

```diff
@@ -82,15 +82,18 @@ sub load_host_tests_docker {
         loadtest 'containers/registry' if is_x86_64;
         loadtest 'containers/docker_compose';
     }
-    loadtest 'containers/validate_btrfs' if (is_x86_64);    # works currently only for x86_64, more are coming (poo#103977)
+    # works currently only for x86_64, more are coming (poo#103977)
+    # Expected to work for all but JeOS on 15sp4 after
+    # https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/13860
+    loadtest 'containers/validate_btrfs' if (is_x86_64 and !(is_jeos && is_sle("=15-SP4")));
 }
 
 
 sub load_container_tests {
     my $runtime = get_required_var('CONTAINER_RUNTIME');
     if (get_var('BOOT_HDD_IMAGE')) {
         loadtest 'installation/bootloader_zkvm' if is_s390x;
-        loadtest 'boot/boot_to_desktop';
+        loadtest 'boot/boot_to_desktop' unless is_jeos;
```

Comment:
Are we sure this is correct? Is this not needed for other container tests?

Reply:
i am quite sure even if i do not like it as it is. I had another proposal some time ago but it was rejected. then all the issue and the complexity with exactly this module was resolved by other PR. for instance the incorporated jobs which needed to have _boot/boot_to_desktop_ and had correlation with jeos have been separated. So it should be fine in the current situation.

Reply:
However, http://duck-norris.qam.suse.de/tests/7970 says otherwise.

Reply:
So, the issue starts to make sense. I confused `boot_to_desktop` with `bootloader_uefi` or `firstrun` which comes from JeOS. I could still not figure out where this comes from, but this one here is now definitely correct.